wiki/Registration/pt-br.md

```diff
@@ -1,43 +1,39 @@
 Como crio uma conta?
 ------------------
 
-<center>
-**<font size="3">Por favor, saiba que possuir mais de uma conta ativa é uma violação das <span class="plainlinks"> \[<http://osu.ppy.sh/wiki/Osu>!:Rules regras do osu!\]</span>. Siga os passos abaixo para configurar a sua conta osu!.**
-</font>
-<span style="font-size:18px">1. Na <span class="plainlinks"> [página principal](https://osu.ppy.sh)</span>, clique no botão "<b>I'm new!</b>" para iniciar o processo de criação de conta. </span>
 
-<span style="font-size:18px">2. Pressione os campos e pressione "Continue" (*Continuar*) </span>
+**Por favor, tenha em mente que ter mais de uma conta infringe as nossas [regras do osu!](/wiki/Rules/). Siga os passos abaixo para criar sua conta no osu!.**
 
-| <b>Legenda</b>                                                                                                                                         |
-|-------------------------------------------------------------------------------------------------------------------------------------------------------|
-| **1. Nome de usuário**                                                                                                                                       |
-| a. Máximo de 15 caracteres                                                                                                                           |
-| b. Use apenas os caracteres alfanuméricos ("a-z, A-Z, 0-9"), subtraços ("\_"), colchetes, ("\[" e "\]") e traços ("-"). |
-| **2. Endereço de e-mail**                                                                                                                                 |
-| **3. Senha**                                                                                                                                       |
-| **4. Aviso**                                                                                                                                           |
-| **5. Botão de continuar**                                                                                                                                |
+1. Na [página principal](https://osu.ppy.sh), Clique em ``Sou novo!`` para iniciar o processo de registro
```

Comment:
adding a space actually breaks the code there, better keep it like this since en.md doesn't have spaces as well